### PR TITLE
Android TV instructions added

### PR DESCRIPTION
**Pull request by puglieri**

### CODE REVIEW

File: youtube-ads.md
Comment (slavaleleka) [inline youtube-ads.md:1]: how is this file related to the android tv instructions added? 

it's placed in the root directory. is it ok?

Comment (puglieri) [reply youtube-ads.md:1]: It's not related, I've used it as a template/base for the new article (the article about YouTube is still there, it was just copy + overwrite)

Comment (el-termikael) [reply youtube-ads.md:1]: I guess it's a mistake
@puglieri youtube-ads.md is the file that must not be in this PR

**Review by el-termikael — CHANGES_REQUESTED** [review]

File: docs/adguard-for-android/solving-problems/android-tv.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+In most cases, stock browsers cannot install a file on your TV, but you can download a browser from the Google Play Store that has this functionality. In our guide, we will consider an example of installation using the TV Bro browser, but there are other options and you can choose the one that better suits your needs. As an alternative, you can use the [Downloader](https://play.google.com/store/apps/details?id=com.esaba.downloader) app.
+
+## Installing AdGuard for Android TV via browser
+
+1. Install the [TV Bro browser](https://play.google.com/store/apps/details?id=com.phlox.tvwebbrowser) on your Android TV.  
+
+1. Download and install AdGuard for Android TV:
+
+* Open the installed TV Bro browser on your Android TV.
```
Comment (el-termikael) [inline docs/adguard-for-android/solving-problems/android-tv.md:20]: Unordered list style, replace asterisks with dashes

File: docs/adguard-for-android/solving-problems/android-tv.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+* Open the terminal or command prompt on your computer.
+* Enter the command `adb connect` and paste the IP address of your TV.
+
+The connection will be established.
+
+1. Install AdGuard for Android TV via ADB:
+
+* In the terminal, enter the command `adb install Downloads/adguard_tv.apk`. If necessary, replace `Downloads/adguard_tv.apk` with your path.
+* Wait for a message in the terminal indicating the successful installation of the app.
```
Comment (el-termikael) [inline docs/adguard-for-android/solving-problems/android-tv.md:61]: a trailing whitespace needs to be added

File: youtube-ads.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+1. Select *AdGuard* from the list of apps.
+
+    ![Share to YouTube step 2 *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/android/youtube/android-youtube-share-step2.png)
+
+That's it! A new window with the video will open where you'll be able to watch it without being interrupted by ads.
+
+## Watch YouTube in a browser
+
+Alternatively, you can also watch YouTube in a browser and there will be no ads if you have AdGuard installed and enabled.
```
Comment (el-termikael) [inline youtube-ads.md:32]: you still need to remove this file from this PR

File: docs/adguard-for-android/solving-problems/adguard-for-android-tv.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+- Open the terminal or command prompt on your computer.
+- Enter the command `adb connect` and paste the IP address of your TV.
+
+The connection will be established.
+
+1. Install AdGuard for Android TV via ADB:
+
+- In the terminal, enter the command `adb install Downloads/adguard_tv.apk`. If
+necessary, replace `Downloads/adguard_tv.apk` with your path.
+- Wait for a message in the terminal indicating the successful installation of the app.
```
Comment (el-termikael) [inline docs/adguard-for-android/solving-problems/adguard-for-android-tv.md:62]: and you still need to add an empty string below

File: docs/adguard-for-android/solving-problems/adguard-for-android-tv.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+- Select *System* → *About*.
+- Find *Network* and select *Wi-Fi* or *Ethernet*, depending on your connection type.
+- Go to the network section and find *IP address*. Note down this IP address.
+
+1. Connect to Android TV via ADB:
+
+- Open the terminal or command prompt on your computer.
+- Enter the command `adb connect` and paste the IP address of your TV.
+
+The connection will be established.
```
Comment (slavaleleka) [inline docs/adguard-for-android/solving-problems/adguard-for-android-tv.md:56]: так и задумано?
https://uploads.adguard.org/sllgfbvfjkp5g.png

**Review by el-termikael — APPROVED** [review]

**Pull request merged**